File: python/ex6.py
# s = [i * i for i in xrange(1, 11)]
n = 100
sum_n = (n * (n + 1) * (2 * n + 1)) / 6
prod_n = n * (n + 1) / 2
print prod_n * prod_n - sum_n

File: src/ex6.rs
pub fn ex6() {
	let n = 100u64;
	let sum_n = (n * (n + 1) * (2 * n + 1)) / 6;
	let prod_n = n * (n + 1) / 2;
	println!("{:?}", prod_n * prod_n - sum_n);
}
